fabfile.py

```diff
@@ -1,28 +0,0 @@
-from fabric.api import local, task, abort, settings
```

Comment:
This is a convenience for pushing to pypi. If removing it, what's the alternative other than doing a lot of manual commands?